share/spice/yoda_speak/yoda_speak.handlebars

```diff
@@ -0,0 +1 @@
+{{data}}
```

Comment:
@thejasbabu similar to what I mentioned in your other PR, let's skip all this custom handlebars and CSS to keep things simple and in-line with our other Instant Answers